spec/controllers/reservations_controller_spec.rb

```diff
@@ -29,8 +30,45 @@
     end
   end
 
+  describe "DELETE #destroy" do
+    context "when user is logged in" do
+      before do
+        sign_in_user
+        delete_destroy(wishlist.id, item.id, reservation.id)
```

Comment:
Any of the test shouldn't be passing. If they do, then there is a bug in our auth logic. Right now, with how you created the models in `let` statements:
```ruby
let(:wishlist) { create(:wishlist) }
let(:item) { create(:item) }
let(:reservation) { create(:reservation) }
```
each of them aren't connected at all. The `wishlist` is OK, but the `item` is *not* OK, because it does *not* belong to the `wishlist` created a line above. When you call `create(:item)` FactoryGirl creates a new, completely different, wishlist internally. The same things happens in the `let(:item)`.